### Chess engine autoplay action server.

# Goal
uint16  num_plies   # maximum number of plies [0 == run to end of game]
float32 delay       # delay between moves [seconds, 0.0 == no delay]
---
# Result
int8    rc          # 0 = ok, <0 = error
---
# Feedback
uint16  ply_cnt     # current count of autoplay plies made
